src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String pattern = "dd.MM.yyyy";
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(pattern);
+    private static final String WHITE_SPASE = " ";
```

Comment:
```suggestion
    private static final String WHITE_SPACE = " ";
```
firstly, it's WHITE_SPACE, but it doesn't make it more informative when we hide white space literal in WHITE_SPACE constant, the constant name should denote its role - name it DELIMITER or SEPARATOR